app/views/home/index.html.haml

```diff
@@ -0,0 +1,16 @@
+= render 'layouts/sorting_bar'
+- if notice
+  %p#notice= notice
+%br
+.food-contents
+  .container-fluid.container.food-table
+    - @products.each_slice(4) do |product|
```

Comment:
that's a `product_slice` not a `product`. Naming matters :)

Reply:
solved ✓